canonical-web-and-design/jaas.ai
# Add instant.page to preload links on hover

## Done

Add instant.page to preload links on hover

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Open your network panel
- Hover over an internal link
- Verify network request for that page
